src/panels/lovelace/views/hui-view.ts

```diff
@@ -54,7 +54,11 @@ export class HUIView extends ReactiveElement {
   // Public to make demo happy
   public createCardElement(cardConfig: LovelaceCardConfig) {
     const element = createCardElement(cardConfig) as LovelaceCard;
-    element.hass = this.hass;
+    try {
+      element.hass = this.hass;
+    } catch (_e: any) {
+      /* ignore */
```

Comment:
Maybe create a helper function that sets `hass` and returns either the element or an error element.

Reply:
I don't know about this... the card may recover on the next hass update, potentially.
...on the other hand it may not recover and keep hogging resources in endless loops... :thinking:

Reply:
It has a bug, so should be fixed 😄 

Better to fail hard so the dev will fix it.